specs/templates/component-interaction-design.md

```diff
@@ -1,36 +1,44 @@
-# Interaction Design (IxD) [Template]
+# Nimble Component (IxD) [Template]
 
 ## Overview
-
-*The name of the component, along with high-level description of what the design includes.*
+*The name of the component, along with high-level description of what the component is used for and it's value to the end-user experience.*
 
 ### Background
-
 *Relevant historical or background information, including:*
-- *Additional client research that isn't captured in linked issues*
-- *Link to Interaction Design issue*
+- *Link to interaction design issue*
 - *Link to relevant work items, related existing issues, etc.*
+- *Link to visual design specification*
+- *Additional client research that isn't captured in linked issues*
 
 ## Usage
+- *When should client-users use this component?*
+- *When should client-users not use this component?*
+
+### Anatomy
+- *What parts make up this component?*
+- *How should client-users use the component parts?*
 
-- *How, when, and where should you use this component?*
-- *Compare this component with other related components. How does it differ?*
-- *How should you decide between related components?*
-- *Should the component support alternate visual styles? (e.g. primary, outline, block, ghost, disabled)*
-- *When should you use alternate visual styles?*
+### Related Components
+- *How does this component differ from other related components?*
+- *How should client-users decide between related components?*
 
 ## Design
+### Configuration
+*What types of options need to be available on the component to support client-user use cases?*
 
-- *Full description of the proposed interaction design*
-- *Include links or embedded views of design in XD or Figma and provide any additional context*
-- *Include accessibility considerations like keyboard interactions and ARIA role/states/properties*
-- *Include mobile considerations like touch interactions and behavior on small screens*
```

Comment:
Is there a risk that eliminating mobile considerations makes future mobile support much more difficult? Or is mobile an expensive distraction?

Reply:
I'm also reluctant to remove the reminder about mobile considerations. I'd at least like us to explicitly say "mobile is out of scope for now" for each component. But even better is to use the reminder to get us to think about low-hanging fruit things we can do to support mobile (thinking about overflow states, avoiding hover-only interactions). I'm hopeful we can use this process to slowly bend our trajectory towards better support with minimal extra effort, but if we remove it then we'll most likely forget about it.